src/utils/leetcode_util.py

```diff
@@ -5,6 +5,7 @@ class LeetcodeUtil:
     GRAPH_URL = "https://leetcode.com/graphql/"
     PROBLEM_URL = "https://leetcode.com/problems/"
     ALL_PROBLEMS_URL = "https://leetcode.com/api/problems/all/"
+    data_limit = 100
```

Comment:
Use `DATA_LIMIT` as all caps convention for class global variables

Reply:
Good spot, thanks!